src/test/java/software/amazon/event/kafkaconnector/mapping/DefaultEventBridgeMapperTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package software.amazon.event.kafkaconnector.mapping;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.kafka.common.record.TimestampType;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import software.amazon.event.kafkaconnector.EventBridgeSinkConfig;
+
+public class DefaultEventBridgeMapperTest {
+
+  @Test
+  @DisplayName("Mapping without timeMapperClass parameter defined should give time null")
+  public void mapWithDeafultTimeMapper() {
```

Comment:
oops, just saw a typo

```suggestion
  public void mapWithDefaultTimeMapper() {
```